docs/README.md

```diff
@@ -28,6 +29,7 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 | AWS FSx for Lustre CSI Driver \ Kubernetes Version| v1.11 | v1.12 | v1.13 | v1.14 | v1.15 |
```

Comment:
probably we should just make the column 1.14+ so we dont have to update it all the time